public/index.html

```diff
@@ -1,22 +1,60 @@
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
-    <head>
-        <meta charset="utf-8">
-        <title>MapGT</title>
-        <link href="style.css" rel="stylesheet">
-    </head>
-    <body>
-        <!-- Rounded switch -->
-        <!-- <script src="../index.js"></script> -->
-        <label class="switch">
-            <input type="checkbox" onclick="toggleFloors()">
-            <span class="slider round"></span>
-        </label>
 
-        <object id="floor1" type="image/svg+xml" data="assets/maps/ICF1Edit.svg"></object>
-        <object id="floor2" type="image/svg+xml" data="assets/maps/ICF2Edit.svg"></object>
+<head>
+    <meta charset="utf-8">
+    <title>MapGT</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Can we get rid of jQuery? We're only using it for `getJSON` which is super easy to do with vanilla JS